content/en/docs/demos/certificate_rotation.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Rotating the MeshRootCertificate"
+description: "Using the OSM CLI to rotate the MeshRootCertificate"
+type: docs
+weight: 22
+---
+
+This guide demonstrates how to use the [MeshRootCertificate](../guides/certificate_management/certificate-rotation.md) preview feature with the OSM cli. 
+
+>  WARNING:  This feature is currently in preview and requires the `EnableMeshRootCertificate` feature flag enabled in the MeshConfig.  
+
+## Prerequisites
+
+- Kubernetes cluster running Kubernetes {{< param min_k8s_version >}} or greater.
+- Have `kubectl` available to interact with the API server.
+- Have `osm` CLI available for installing and managing the service mesh.
+- Have `openssl` avalaible to view certificate information
+
+## Demo
+
+The following demo shows how to use the osm cli to initiate a root certificate rotation.  Learn more about this process in the [Certificate Rotation documentation](../guides/certificate_management/certificate-rotation.md).
+
+1. Install OSM with the `EnableMeshRootCertificate` feature flag set to true.
+   ```console
+   osm install --set="osm.featureFlags.enableMeshRootCertificate=true"
+   ```
+
+1. Check that the default `MeshRootCertificate` was created:
+   ```console
+   kubectl get mrc -n osm-system 
+   NAME                          ROLE
+   osm-mesh-root-certificate     active
+   ```
+
+   Confirm that a Secret was created with the Root certificate:
+   ```
+   kubectl get secrets -n osm-system
+   NAME              TYPE     DATA   AGE
+   osm-ca-bundle     Opaque   2      18h
+   ```
+
+   Take note of the modulus of the original certificate:
+   ```
+   kubectl get secret -n osm-system  osm-ca-bundle -o jsonpath='{.data.ca\.crt}' |
+    base64 -d |
+    openssl x509 -modulus -noout
+    Modulus=BC79C8202E8EE11DC27335E7313BB9466DEC870367F57E9BA56F1E83296B9872FCDB924DC2A9B8C012EA72F7F76356E895D56A9D2DFAF8A62E768156A81F3787A526CAA0858FD1BC854E4BFE2D27C10F7DCC4586B7AB71572406A5A9648AB317C77DF56ED74F970A007D95FD11C1A51170A2579B3AE7CFE257714611A0C1FE11D2F8F3A6499CA94536B18E751D013B283A4331C1E6830B834CEC14A2A58B11537E2AFDB2A01BD5F606237F2DB25387F2954161AF0EC99D377D0CD3813B61BE39DF427C5B412D38207A0744E4B40869294B05171E1CE45253B8B4BEA15C009D8AF3DE7832E12D668636CF2D13D8F363A79F53CEECA4B3718FE557E136B6162E27
+   ```
+
+1. Enable permissive traffic policy mode to set up automatic application connectivity.
+    > Note: this is not a requirement to use `MeshRootCertificate`'s but simplifies the demo by not requiring explicit traffic policies for application connectivity.
+
+    ```bash
+    kubectl patch meshconfig osm-mesh-config -n "$osm_namespace" -p '{"spec":{"traffic":{"enablePermissiveTrafficPolicyMode":true}}}'  --type=merge
```

Comment:
The spacing looks off here in the deploy preview
https://deploy-preview-448--osm-docs.netlify.app/docs/demos/certificate_rotation/

Reply:
fixed I believe